### ISSUE TITLE
Pie chart using altair

### ISSUE THREAD

**Issue by yupenghegh** [altair-viz/altair#1098]
Hi, I am wondering if there is a way to plot pie chart using altair. Thanks! 

Sorry if it is a trivial issue but I am unable to find any documentation or example about it. 

Yupeng

**Comment by Alcampopiano** [comment]
I do not believe pie charts are directly supported in Vega-Lite or Altair. However, there is a neat hack that might work for you [here](https://github.com/vega/vega-lite/issues/408). You could also use Vega.

**Comment by iliatimofeev** [comment]
Please upvote [VL ER](https://github.com/vega/vega-lite/issues/408) by placing a thumb if you need that functionality

**Comment by domoritz** [comment]
Pie charts are now supported in VL 4.9 so they will come to Altair as well.
